app/index.js

```diff
@@ -395,7 +395,8 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       createWindow,
       updatePlugins: () => {
         plugins.updatePlugins({force: true});
-      }
+      },
+      getPlugins: plugins.getModules
```

Comment:
`getLoadedPlugins` will be a better name and it will export only plugin name and version. Not the entire module.